cmd/minikube/cmd/ip.go

```diff
@@ -51,7 +51,7 @@ var ipCmd = &cobra.Command{
 		if err != nil {
 			exit.WithError("Error getting IP", err)
 		}
-		console.Out(ip)
+		out.Out(ip)
```

Comment:
Not your fault, but while you are here, this should actually be `out.Ln`. We accidentally broke compatibility with the translation: something I noticed when refactoring TestRegistry.

Reply:
Fixed.